components/hab/src/command/pkg/delete.rs

```diff
@@ -39,6 +41,9 @@ pub fn start(ui: &mut UI,
 
     if let Err(err) = api_client.delete_package((ident, target), token) {
         println!("Failed to delete '{}': {:?}", ident, err);
+        if let api_client::Error::APIError(StatusCode::NotFound, _) = err {
+            println!("Perhaps you may need to specify a platform target argument");
```

Comment:
I realize this is nitpicky, but this sentence reads a little awkwardly for me.  I'd change it to either:
`You may need to specify a platform target argument`
or
`Perhaps you need to specify a platform target argument`

And also the 2 instances below.

Reply:
ok - I like the first option, will update